themes/theme-gmd/pages/Product/components/Media/components/ProductImageSlider/index.jsx

```diff
@@ -70,7 +70,12 @@ class ProductImageSlider extends Component {
     const {
       product, images, 'aria-hidden': ariaHidden, className,
     } = this.props;
-    const { HeroImage: pdpResolutions } = getThemeSettings('AppImages') || {};
+    const {
+      HeroImage: pdpResolutions = [{
```

Comment:
Its not smart way. Please set defaults on appWillStart to Configuration and use runtime configuration everywhere

Reply:
changed